fred2/management.cpp

```diff
@@ -2082,9 +2087,9 @@ int orders_reference_handler(int code, char *msg)
 
 	ShipGoalsDlg dlg_goals;
 
-	n = code & SRC_DATA_MASK;
-	switch (code & SRC_MASK) {
-		case SRC_SHIP_ORDER:
+	n = source_index;
```

Comment:
Well, this is now obsolete. n is just source index, so just using source_index directly is cleaner here

Reply:
Well, I didn't want to make the diff any more complicated...